backend/spring-routie/src/main/java/routie/routie/infrastructure/routievalidator/ValidatorConfig.java

```diff
@@ -0,0 +1,17 @@
+package routie.routie.infrastructure.routievalidator;
+
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import routie.routie.domain.routievalidator.RoutieValidator;
+
+@Configuration
+public class ValidatorConfig {
```

Comment:
이 부분도 마찬가지입니다.

전체적으로 체크 부탁드립니다
```suggestion
public class RoutieValidatorConfig {
```

Reply:
2개 부분 모두 수정하겠습니다